_includes/glossary/README.md

```diff
@@ -0,0 +1,38 @@
+# Kubernetes Glossary
+
+To write a glossary snippet, start with a copy of the template in this directory, `_glossary-template.md`. Make sure to provide (or omit) values for the following fields:
+
+* (Required) `term`
+* (Optional) `formerly` and `related`. If you do not provide these values, remove the fields.
+* (Required) short description. Make sure to replace the instructional text in the template with your content.
+* (Optional) long description. If you do not provide a long description, remove the `{% capture %}` and `{% endcapture %}` tags together with the instructional text.
+
+The template also contains basic information about how to write your snippet. For additional guidance, read on in this readme.
+
+## Glossary snippet style guide
+
+This style guide supplements the guidance provided in the glossary template. It's intended to help you think about what and how to write glossary definitions. For more general guidance on style, consult [the core docs style guide](https://kubernetes.io/docs/home/contribute/style-guide/), also available in this repo at `docs/home/contribute/style-guide.md`.
+
+### Minimum viable snippet:
+
+Every snippet must include at least the short description. The long description is optional, but should be provided for terms that need additional clarification.
+
+<short-description>Required. One line (or two short lines) that provides a minimum definition. Do not repeat the term. Prefer fragments. Model after tooltips. End with a period.
+
+<long-description>Optional. Longer additional text to appear after (in conjunction with) short description. Provide in cases where the short description is not sufficient for the intro paragraph to a topic. Write complete but concise sentences.
+
+### Examples
+
+<term>Pod
```

Comment:
Do you want the angle brackets to be visible? If so, I think you have to use the escaped versions (&lt; and &gt;) since otherwise the parser thinks it's HTML
(Not sure if that'll help with the newlines but currently those aren't showing)